Makefile

```diff
@@ -72,6 +72,10 @@ update-pins: update-libcalico-pin
 ###############################################################################
 # Building the binary
 ###############################################################################
+
+# Reassign VALIDARCHES since for calicoctl we do not want to exclude any builds.
+VALIDARCHES=$(patsubst Dockerfile.%,%,$(wildcard Dockerfile.*))
```

Comment:
This needs to be above the include for the common makefile otherwise the targets in the common makefile won't be created for all the valid arches